From bcabdd0ba9753cbad0d81b6ef85775db7ee4d393 Mon Sep 17 00:00:00 2001
From: Vladan Popovic <vpopovic@redhat.com>
Date: Fri, 7 Mar 2014 17:00:47 +0100
Subject: [PATCH] Revert "Replace oslo.sphinx with oslosphinx"

This reverts commit c926216a9bd64d5cdd9731bd34e03613940d32e9.

Conflicts:
	test-requirements.txt
---
 doc/README.rst        | 2 +-
 doc/source/conf.py    | 2 +-
 test-requirements.txt | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/doc/README.rst b/doc/README.rst
index dbaaa40..92db83b 100644
--- a/doc/README.rst
+++ b/doc/README.rst
@@ -12,7 +12,7 @@ environment, such as the virtualenv created by ``run_tests.sh`` or
 ``tools/install_venv.py``.  A leaner but sufficient environment can be
 created by starting with one that is suitable for running Nova (such
 as the one created by DevStack) and then using pip to install
-oslosphinx.
+oslo.sphinx.
 
 Building the docs
 -----------------
diff --git a/doc/source/conf.py b/doc/source/conf.py
index 4735fcb..43e972e 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -33,7 +33,7 @@ extensions = ['sphinx.ext.autodoc',
               'sphinx.ext.pngmath',
               'sphinx.ext.ifconfig',
               'sphinx.ext.graphviz',
-              'oslosphinx',
+              'oslo.sphinx',
               ]
 
 todo_include_todos = True
diff --git a/test-requirements.txt b/test-requirements.txt
index 8a8079f..e7f644b 100644
--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -10,6 +10,6 @@ psycopg2
 pylint==0.25.2
 python-subunit>=0.0.18
 sphinx>=1.1.2,<1.2
-oslosphinx
+oslo.sphinx
 testrepository>=0.0.18
 testtools>=0.9.34
